errors/index.js

```diff
@@ -1,8 +1,17 @@
 module.exports = {
 
-  //This middleware is intended invoked after the router has handled all possible urls 
+  handlePSQLErrors: function(err, req, res, next) {
+    if (err.code === '22P02') {
+      next({status: 400, msg: 'Bad request, expected numeric id'})
+    } else {
+      next(err);
+    }
+  },
+
   handle400StatusCodes: function(err, req, res, next) {
```

Comment:
I would rename this function to be handleCustomErrors as it will handle more than 400 :)